contents.tex

```diff
@@ -0,0 +1,73 @@
+\mode*
+
+\section{Introduction}%
+\label{Introduction}
+
+In this lab we will evaluate some tools for private communication, in 
+particular, email and instant messaging.
+
+Email security has the longest history, this has been around since the 80s.
+Have they gained wide-spread adoption yet?
+No.
+There are two competing approaches: Pretty Good Privacy (PGP) and S/MIME 
+(comparable to HTTPS for websites).
+But there are alternatives coming, \eg ProtonMail made the headlines a few 
+years ago.
+
+The security of instant messaging looks better than that of email.
+This is largely due to the extensive use of smartphones for messaging.
+There are apps such as Signal, WhatsApp (uses Signal's protocol) and a few 
+more.
+Have they gained wide-spread adoption yet?
+More than for email.
+
+
+\section{Assignment}%
+\label{sec:Tasks}
+
+The assignment is divided into two parts, the first focuses on email and the 
+second on instant messaging.
+
+\subsection{Email security}
+
+List the security (and privacy) expectations that you have on email.
+Try to install and use a tool (\eg GPG, ProtonMail, \etc) that achieves those 
```

Comment:
Is ProtonMail a tool or a Service?

Reply:
Service. But you can install some tools to use it too (e.g. their app on the phone).